Zend/zend_object_handlers.c

```diff
@@ -1538,6 +1571,17 @@ ZEND_API void zend_std_unset_property(zend_object *zobj, zend_string *name, void
 		if (!zobj) {
 			return;
 		}
+
+		if (UNEXPECTED(guard)) {
+			guard = zend_get_property_guard(zobj, name);
+			if (!((*guard) & IN_ISSET)) {
```

Comment:
Should this not be `IS_UNSET`? Zend/tests/lazy_objects/gh18038-010.phpt looks wrong.

Reply:
Good catch! Thank you